Documentation/reproducible-research.rst

```diff
@@ -131,3 +131,39 @@ SeisSol setups for community benchmark are described in the cookbook  (see :ref:
 
 .. include:: table_cookbook.rst
 
+
+We provide tutorial materials designed for software training, e.g., CHEESE Advanced training (https://www.hlrs.de/training/2021-10-19-cheese/) and ICTP Advanced Workshop on Earthquake Fault Mechanics (https://indico.ictp.it/event/8715/overview).  The training material depends on an easy-to-obtain and uniform installation of SeisSol (Docker version) and related tools (e.g. Gmsh, Paraview) and necessary input files. 
```

Comment:
I would write the link like:
`CHEESE Advanced training <CHEESE Advanced training>`_
and same for https://indico.ictp.it/event/8715/overview